features/eid_tmtrainer.lua

```diff
@@ -53,6 +54,7 @@ function EID:CheckGlitchedItemConfig(id)
 	
 	local localizedNames = EID:getDescriptionEntry("GlitchedItemText")
 	local localizedNamesEnglish = EID:getDescriptionEntryEnglish("GlitchedItemText")
+	local pluralize = EID:getDescriptionEntry("Pluralize")
```

Comment:
Languages are very complicated, so it would be best, if Pluralize would work like a function that can be defined for each language.

Example:
In German, nouns have genders and based on that, the pluralization method changes.

1 Heart = 1 Herz ; 2 Hearts = 2 Herzen
1 Bomb = 1 Bombe ; 2 Bombs = 2 Bomben
1 Key = 1 Schlüssel; 2 Keys = 2 Schlüssel